### PR TITLE
Validation section user manual

### PR DESCRIPTION
**Pull request by nitbharambe**

### CODE REVIEW

**Comment by TonyXiang8787** [comment]
@nitbharambe  the example notebooks are all removed. You can also remove in this PR.

File: docs/user_manual/model-validation.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Model validation
+
+power-grid-model is validated using multiple cases present in [\tests\data](..\..\tests\data) folder.
```
Comment (TonyXiang8787) [inline docs/user_manual/model-validation.md:9]: link not working

File: docs/user_manual/model-validation.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+<!--
+SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# Model validation
+
+power-grid-model is validated using multiple cases present in [/tests/data](../../tests/data) folder.
```
Comment (TonyXiang8787) [inline docs/user_manual/model-validation.md:9]: relative links to data folder will not work. maybe just use absolute http url

File: docs/user_manual/model-validation.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+The cases of a differences in modelling between both the libraries are handled by theoretical workarounds.
+For example in power-grid-model, source impedance is included for all component sequences. 
+In pandapower, source impedance is present only in positive sequence network whereas it considered in all sequence components in power-grid-model.
+Source impedance is then set to a low value to match this modelling difference.
+Hence, the result of source component here should be ignored.
+The output result attributes of power-grid-model are validated at a tolerance value of $\pm10^{-5}$ of respective unit.
+Both the iterative algorithms: Newton Raphson and Iterative current are validated.
+
+All the test cases can be found in [/tests/data/power_flow/pandapower](../../tests/data/power_flow/pandapower).
```
Comment (TonyXiang8787) [inline docs/user_manual/model-validation.md:25]: same here, relative link to outside docs folder will not work

File: docs/user_manual/model-validation.md (hunk 2 above)
Comment (nitbharambe) [reply docs/user_manual/model-validation.md]: Fixed them

**Review by TonyXiang8787 — APPROVED** [review]

**Pull request merged**